README.md

```diff
@@ -1,30 +1,49 @@
-# Installation
+# Read Me A Story
+
+Read Me A Story is an app that allows family members record, upload and playback bedtime stories for young children in the family when they are unable to read them in person.
+
+A parent can sign up to the app, create a family group and invite their partner or relatives to join. Family members can then choose to record and upload stories, or view a list of stories created by the family. Stories can be selected from the list and played back to the child.
+
+Read Me A Story was created by Minimum Viable Panic as their final project in the [Northcoders](https://northcoders.com) coding bootcamp. To find out more about the project check out the [product demonstration](#) or head on over to the [live app](#).
+
```

Comment:
should we have a link to the front end repo too?

Reply:
Have reworded this section to include link